src/pybind/mgr/orchestrator_cli/run-tox.sh

```diff
@@ -40,7 +40,7 @@ ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
 # use bash string manipulation to strip off any trailing comma
 ENV_LIST=${ENV_LIST%,}
 
-tox -c "${TOX_PATH}" -e "${ENV_LIST}"
+tox -c "${TOX_PATH}" -e "${ENV_LIST}" "@"
```

Comment:
s/@/$@/

Reply:
ouch! thanks for catching that, @tchaikov